xbmc/pvr/PVRManager.cpp

```diff
@@ -493,8 +493,11 @@ void CPVRManager::Unload()
 
 void CPVRManager::Shutdown()
 {
+// quick and dirty Krypton-only fix for http://trac.kodi.tv/ticket/17374
+#if 0
   // set system wakeup data
   SetWakeupCommand();
```

Comment:
Why? I just was in the mood. And as this is krypton-only quick and dirty I don't really care, tbh. I don't think I will change this. I don't want to spend more time on this.